packages/host/app/lib/current-run.ts

```diff
@@ -349,6 +349,16 @@ export class CurrentRun {
       return invalidationList;
     }
 
+    // Check for deleted files - files that exist in index but not on filesystem
```

Comment:
I made a concious decision to add deletedUrls to the invalidation list. Although I was told to tombstone everything inside of the db. 

Reason being is batchUpdates fundamentally apply to invalidation array. That way I can be confident that I only tombstone the working index and the batch update gets promotes to boxel_index for me, otherwise more machinery needed. Also confine tombstoning to the `invalidate` function

Reply:
i think that's perfectly fine